contracts/src/StakeTable.sol

```diff
@@ -97,18 +110,11 @@ contract StakeTable is AbstractStakeTable {
         return 0;
     }
 
-    /// @notice Total stakes of the registered keys in the latest stake table (Head).
-    /// @dev Given that the current implementation does not support restaking, the second value of
-    /// the output is set to 0.
-    /// @return The total stake for native token and restaked token respectively.
-    function totalStake() external view override returns (uint256, uint256) {
-        return (totalNativeStake, totalRestakedStake);
-    }
-
     /// @notice Look up the balance of `blsVK`
     /// @param blsVK BLS public key controlled by the user.
     /// @return Current balance owned by the user.
-    function lookupStake(BN254.G2Point memory blsVK) external view override returns (uint64) {
+    /// TODO modify this according to the current spec
+    function lookupStake(BN254.G2Point memory blsVK) external view override returns (uint256) {
```

Comment:
now that a validator is identified by the `stakingAddr`, `blsKey` and `schnorrKey`. Do we want functions like `lookupStake()`, `lookupNode()` and `withdrawFunds()` to use the `stakingAddr` as a parameter? @sveitser @tbro